roles/foreman_provisioning/tasks/configure_ubuntu_24_04.yml

```diff
@@ -1,19 +1,19 @@
-- name: 'create Ubuntu 17.10'
+- name: 'create Ubuntu 24.04'
   shell: >
-    {{ foreman_provisioning_hammer }} os info --title "Ubuntu 17.10" ||
+    {{ foreman_provisioning_hammer }} os info --title "Ubuntu 24.04" ||
     {{ foreman_provisioning_hammer }} os create
-    --name Ubuntu --major 17 --minor 10 --release-name 'artful' --architectures x86_64 --family 'Debian' --media 'Ubuntu mirror' --partition-tables 'Preseed default'
+    --name Ubuntu --major 24 --minor 04 --release-name 'artful' --architectures x86_64 --family 'Debian' --media 'Ubuntu mirror' --partition-tables 'Preseed default'
```

Comment:
Didn't this change to using `24.04` as major?

Reply:
It did, changed